lib/get-file-paths.js

```diff
@@ -4,7 +4,7 @@ var path = require('path');
 
 /**
  * Returns an object of file path data
- * 
+ *
  * @param {object}  file  a vinyl file object
  */
 exports.getFilePaths = function (file) {
```

Comment:
This is another one of those files that is just wayyyy too much magic and a bit complicated.

We need to either fix the structure or something. We can't be splitting and slice path. This feels very sensitivity and can easily break.